front/types/extract.ts

```diff
@@ -35,6 +36,8 @@ export type EventSchemaPropertiesTypeForModel = {
 
 export type ExtractedEventType = {
   id: ModelId;
+  sId: string;
+  schemaId: ModelId;
```

Comment:
schema: EventSchemaType

Reply:
@spolu I removed it here: https://github.com/dust-tt/dust/pull/1169/commits/b795cbddf5d4f48f3e001c93f114db1dc8170ff1